pkg/release/release.go

```diff
@@ -107,15 +104,29 @@ func New(config Config) (*Release, error) {
 		}
 	}
 
+	var fileLogger *filelogger.FileLogger
+	{
+		c := filelogger.Config{
+			K8sClient: k8sClient,
+			Logger:    c.Logger,
+		}
+
+		fileLogger, err = filelogger.New(c)
+		if err != nil {
+			return nil, microerror.Mask(err)
+		}
+	}
```

Comment:
This is cool for me that using internal package enforced simplified external setup.
I didn't even know that filelogger is imported in dependent projects.